metadata-ingestion/src/datahub/ingestion/source/sql/mssql/source.py

```diff
@@ -546,8 +564,8 @@ def _get_procedure_code(
                 code_list.append(row["Text"])
                 if code_slice_text in re.sub(" +", " ", row["Text"].lower()).strip():
                     code_slice_index = index
-            definition = "\n".join(code_list[:code_slice_index])
-            code = "\n".join(code_list[code_slice_index:])
+            definition = "".join(code_list[:code_slice_index])
+            code = "".join(code_list[code_slice_index:])
```

Comment:
This unnecessarily added \n in middle of identifier names. Multi-line procedure response already contains \n character explicitly, so this additional separator is not needed.